libraries/bot-applicationinsights/src/test/java/com/microsoft/bot/applicationinsights/BotTelemetryClientTests.java

```diff
@@ -180,4 +177,13 @@ public void trackPageViewTest() {
             Assert.assertEquals(0, Double.compare(0.6, pageViewTelemetry.getMetrics().get("metric")));
         }).send(Mockito.any(PageViewTelemetry.class));
     }
+
+    @Test
+    public void flushTest() {
+        botTelemetryClient.flush();
+
+        Mockito.verify(mockTelemetryChannel, invocations -> {
```

Comment:
This is just a question, here we are checking against `mockTelemetryChannel` but the one that we need to check if the client right?

Check [JS](https://github.com/microsoft/botbuilder-js/blob/main/libraries/botbuilder-applicationinsights/tests/telemetryClient.test.js#L103).

Reply:
Here I took as a reference the previous unit tests we migrated which are using the `mockTelemetryChannel`. In JS almost all the unit tests are using the client.

Reply:
Ack